syncer/ddl.go

```diff
@@ -138,6 +154,29 @@ func (s *Syncer) splitAndFilterDDL(
 
 		// get real tableNames before apply block-allow list
 		if s.onlineDDL != nil {
+			if onlineDDLFinish && len(tables) > 1 {
```

Comment:
can we abstract this part of logic into a named function, too much indent levels already, hard to read